src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -27,15 +28,22 @@ export default class AnnotationColumnFormatter
 
         if (copyNumberData)
         {
-            let oncoKbIndicator: IndicatorQueryResp|undefined;
-
-            if (oncoKbData && oncoKbData.result && oncoKbData.status === "complete") {
-                oncoKbIndicator = AnnotationColumnFormatter.getIndicatorData(copyNumberData, oncoKbData.result);
+            let oncoKbIndicator: IndicatorQueryResp|undefined = undefined;
+            let oncoKbStatus:"complete"|"pending"|"error" = "complete";
+            const oncoKbGeneExist = !!oncoKbAnnotatedGenes[copyNumberData[0].entrezGeneId];
```

Comment:
i think you could use "in" operator.  slightly more semantic.  minor quibble

Reply:
do we have a good Set object?